networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -0,0 +1,231 @@
+use std::{sync::Arc, time::Duration};
+
+use anyhow::Context;
+use monza_executor::{
+    MonzaExecutor,
+    ExecutableBlock,
+    HashValue,
+    FinalityMode,
+    Transaction,
+    SignatureVerifiedTransaction,
+    SignedTransaction,
+    ExecutableTransactions,
+    v1::MonzaExecutorV1,
+};
+use m1_da_light_node_client::*;
+use async_channel::{Sender, Receiver};
+use sha2::Digest;
+use crate::*;
+use tokio_stream::StreamExt;
+use tokio::sync::RwLock;
+use movement_types::Block;
+
+
+#[derive(Clone)]
+pub struct MonzaPartialFullNode<T : MonzaExecutor + Send + Sync + Clone> {
+    executor: T,
+    transaction_sender : Sender<SignedTransaction>,
+    pub transaction_receiver : Receiver<SignedTransaction>,
+    light_node_client: Arc<RwLock<LightNodeServiceClient<tonic::transport::Channel>>>,
+}
+
+impl <T : MonzaExecutor + Send + Sync + Clone>MonzaPartialFullNode<T> {
```

Comment:
```suggestion
impl <T : SuzukaExecutor + Send + Sync + Clone>SuzukaPartialFullNode<T> {
```

Reply:
Also it looks like proper formatting is not enforced in the workspace, which it should be. I'll look into adding a CI check.